src/plugins/MamicoCoupling.h

```diff
@@ -0,0 +1,62 @@
+#ifndef SRC_PLUGINS_MAMICOCOUPLING_H_
+#define SRC_PLUGINS_MAMICOCOUPLING_H_
```

Comment:
```suggestion
#pragma once
```
(also remove the `#endif` at the bottom of the file.)
In general `pragma once` > include guards